src/main/topLevelPages/myDashboard/DashboardContext.tsx

```diff
@@ -9,6 +10,8 @@ const enum DashboardViews {
 interface DashboardContextProps {
   activityEnabled: boolean;
   setActivityEnabled: (item: boolean) => void;
+  openedDialog: DashboardDialog | undefined;
```

Comment:
Let's follow the `is` convention and convert `openedDialog` to `isOpen` or `isDialogOpen`.